### PR TITLE
[RM-13847] Using full paths to executables

### PR DESCRIPTION
**Pull request by mazebuhu**
Refactored out some direct executable calls in "osd.py". 

Should we do that for all executable calls in ceph-deploy?

For tracker see: http://tracker.ceph.com/issues/13847

### CODE REVIEW

**Comment by alfredodeza** [comment]
ok to test

File: ceph_deploy/osd.py [hunk 1]
```diff
@@ -424,10 +427,11 @@ def disk_zap(args):
         if distro.normalized_name.startswith(('centos', 'red')):
             LOG.info('calling partx on zapped device %s', disk)
             LOG.info('re-reading known partitions will display errors')
+            partx_executables = system.executable_path(distro.conn, 'partx')
```
Comment (alfredodeza) [inline ceph_deploy/osd.py]: why is this `partx_executables` (plural) ?

Comment (mazebuhu) [inline ceph_deploy/osd.py]: It's a typo. Sorry.

**Comment by mazebuhu** [comment]
Ok I fixed the typo.

**Pull request merged**